src/views/catalog/templatescatalog/components/TemplatesCatalogDrawer/utils/helpers.ts

```diff
@@ -0,0 +1,22 @@
+import { StorageClassModel } from '@kubevirt-ui/kubevirt-api/console';
+import { IoK8sApiStorageV1StorageClassList } from '@kubevirt-ui/kubevirt-api/kubernetes';
+import { V1VirtualMachine } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { isDefaultStorageClass } from '@kubevirt-utils/components/DiskModal/DiskFormFields/utils/helpers';
+import { getDataVolumeTemplates } from '@kubevirt-utils/resources/vm';
+import { isEmpty } from '@kubevirt-utils/utils/utils';
+import { k8sGet } from '@openshift-console/dynamic-plugin-sdk';
+
+export const hasStorageClass = async (vm: V1VirtualMachine) => {
```

Comment:
please change the name of the function, From the code I can see that if not all dvt has sc it will fetch and return true if default sc exists. so maybe name it shouldReplaceToTemplateStorageClass (not one of my best) or something better?